test/e2e/nodeportlocal_test.go

```diff
@@ -409,7 +409,8 @@ func NPLTestPodAddMultiPort(t *testing.T, data *TestData) {
 	clientNode := nodeName(0)
 	serverNode := nodeName(1)
 	if len(clusterInfo.windowsNodes) != 0 {
```

Comment:
Is it better to check `len(clusterInfo.windowsNodes) > 1` since you actually need to access `clusterInfo.windowsNodes` after the change?

Reply:
Updated.